markdown/reference/guc/parameter_definitions.html.md.erb

```diff
@@ -2147,6 +2149,14 @@ Identifies the port on which the HAWQ Ranger Plug-in Service runs. The `hawq_rps
 |-------------------------------------------------------------------------|---------------------|-------------------------|
 | valid port number | 8432 | master, reload |
 
+## <a name="hawq_rps_check_local_interval"></a>hawq\_rps\_check\_local\_interval
+
+When HAWQ Ranger authentication high availability mode is enabled and the Ranger Plug-in Service is active on the standby master node, HAWQ attempts to switch back to the service located on the master node as soon as it becomes available. The HAWQ master periodically attempts to re-establish contact with the service on the local node, using `hawq_rps_check_local_interval` as the polling time interval (in seconds) for this contact.
```

Comment:
This wording is a bit unclear as to whether it's talking about just the standby ranger service being used (which is the intent) vs whether the entire master node is down and the standby plug-in is being used.  I think it would be better to repeat the wording used in `ranger-ha.html`, which starts "Should the HAWQ master node fail to communicate with the local Ranger Plug-in Service..."